bucket/registryfinder.json

```diff
@@ -0,0 +1,55 @@
+{
+    "version": "2.52.0.0",
+    "description": "Registry Finder is an improved replacement for the built-in Windows registry editor",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://registry-finder.com/License.txt"
+    },
+    "homepage": "https://registry-finder.com/",
+    "notes": "Run \"Registry Finder\" from the Start Menu, or \"registryfinder.exe\" from the command line.",
+    "architecture": {
+        "64bit": {
+            "url": "https://registry-finder.com/bin/2.52.0.0/RegistryFinder64.zip",
+            "hash": "f57b29e1f6f6f03af05332ce6897c1234a1cf580c8b481f27a689765acc4251d"
+        },
+        "32bit": {
+            "url": "https://registry-finder.com/bin/2.52.0.0/RegistryFinder.zip",
+            "hash": "d8255947a1d7e3be07cdc9021c945afc2d67d7ec5149c5262d05ad044878695a"
+        }
+    },
+    "bin": [
+        [
+            "registryfinder.exe",
+            "registryfinder",
+            "--dataFolder $persist_dir"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "RegistryFinder.exe",
+            "Registry Finder",
+            "--dataFolder $persist_dir"
+        ]
+    ],
+    "checkver": {
+        "re": "bin/([\\d.]+)/"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://registry-finder.com/bin/$version/RegistryFinder64.zip",
+                "hash": {
+                    "url": "https://registry-finder.com/",
+                    "find": "Archive 64-bit</a> SHA-256: $sha256"
+                }
+            },
+            "32bit": {
+                "url": "https://registry-finder.com/bin/$version/RegistryFinder.zip",
+                "hash": {
+                    "url": "https://registry-finder.com/",
+                    "find": "Archive 32-bit</a> SHA-256: $sha256"
```

Comment:
```suggestion
                    "regex": "Archive 32-bit</a> SHA-256: $sha256"
```

Reply:
I was just using the examples from the wiki.  Maybe the wiki should be updated.